frontend/src/app/tv/tv.service.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * The TV Service abstracts HTTP requests to the backend
+ * from the components.
+ *
+ * @author Will Zahrt
+ * @copyright 2024
+ * @license MIT
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, map } from 'rxjs';
+import {
+  ArticleDraft,
+  ArticleOverview,
+  ArticleOverviewJson,
+  parseArticleOverviewJson
+} from '../welcome/welcome.model';
+import {
+  DEFAULT_PAGINATION_PARAMS,
+  PaginationParams,
+  Paginator
+} from '../pagination';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TvService {
```

Comment:
Assume this file is not relevant to this PR anyway, but we can use `Signage` throughout in the frontend as well!